pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -59,6 +59,9 @@ var (
 		# gather information with a specific local folder to copy to
 		  oc adm must-gather --dest-dir=/local/directory
 
+		# gather audit information
+		  oc adm must-gather -- /usr/bin/gather_audit_logs
```

Comment:
Will `oc adm must-gather -- gather_audit_logs` work as well?

Reply:
`./oc adm must-gather -- gather_audit_logs` produces:
```
$ ls must-gather.local.618611527624369846/registry-svc-ci-openshift-org-ocp-4-6-2020-09-26-142601-sha256-c5b27546b5bb33e0af0bdd7610a0f19075bb68c78f39233db743671b9f043f6b/audit_logs/
kube-apiserver                     oauth-apiserver                     openshift-apiserver
kube-apiserver.audit_logs_listing  oauth-apiserver.audit_logs_listing  openshift-apiserver.audit_logs_listing
```

Reply:
Must gather is generating a pod with `/usr/bin/gather`. `/usr/bin/gather_audit_logs` is more aligned with that than just `gather_audit_logs`.